roles/patch_host_config/tasks/main.yml

```diff
@@ -93,3 +93,32 @@
           ]
         }
       register: http_reply
+
+- name: Load patch for worker partition
+  when: hostvars[host_name]['partitions'] is defined
+  block:
+    - name: Template patch partition
+      set_fact:
+        worker_partition: "{{ lookup('template', 'patch-worker-partition.j2') }}"
```

Comment:
The template is going to need to account for the configuration in `hostvars[host_name]['partitions']`, as it's currently hard-coded to 10x1GiB partitions, which is not going to work for most cases.